### ISSUE TITLE
Add support of popular status lines

### ISSUE THREAD

**Issue by semanser** [semanser/vim-outdated-plugins#5]

**Comment by YodaEmbedding** [comment]
Also just the normal built-in vim statusline. Maybe do this by asynchronously updating some global variable at startup?

EDIT: Looks like you already do that... `g:needToUpDate`

-----

Also, shouldn't statusline functionality be responsibility of the end user?

**Comment by semanser** [comment]
@SicariusNoctis Yes, it's optional, but if this can be implemented, why this shouldn't work out of the box?)
